src/Graph.ts

```diff
@@ -382,6 +382,25 @@ export class Graph {
     );
   }
 
+  /**
+   * async promise, allows developer to assign people to task
+   *
+   * @param {string} taskId
+   * @param {string} eTag
+   * @param {*} people
+   * @returns {Promise<any>}
+   * @memberof Graph
+   */
+  public async planner_assignPeopleToTask(taskId: string, eTag: string, people: any): Promise<any> {
```

Comment:
Can you make the order of the parameters same as _setTaskDetails so it is consistent?

Reply:
Yup: c1b04e8